Documentation/Usage.md

```diff
@@ -197,20 +199,22 @@ let parameters: [String: [String]] = [
     "baz": ["a", "b"],
     "qux": ["x", "y", "z"]
 ]
+
 // All three of these calls are equivalent
 AF.request("https://httpbin.org/post", method: .post, parameters: parameters)
 AF.request("https://httpbin.org/post", method: .post, parameters: parameters, encoder: URLEncodedFormParameterEncoder.default)
 AF.request("https://httpbin.org/post", method: .post, parameters: parameters, encoder: URLEncodedFormParameterEncoder(destination: .httpBody))
+
 // HTTP body: "qux[]=x&qux[]=y&qux[]=z&baz[]=a&baz[]=b&foo[]=bar"
 ```
 
 ##### Configuring the Encoding of `Array` Parameters
 
-Since there is no published specification for how to encode collection types, by default Alamofire follows the convention of appending `[]` to the key for array values (`foo[]=1&foo[]=2`), and appending the key surrounded by square brackets for nested dictionary values (`foo[bar]=baz`).
+Since there is no published specification for how to encode collection types, by default, Alamofire follows the convention of appending `[]` to the key for array values (`foo[]=1&foo[]=2`), and appending the key surrounded by square brackets for nested dictionary values (`foo[bar]=baz`).
```

Comment:
This comma doesn't seem necessary. Adding one seems to make the clause part of the preceding statement.

Reply:
Removed in c5ec5f7c.